homeassistant/components/intellifire/light.py

```diff
@@ -95,3 +94,4 @@ async def async_setup_entry(
             for description in INTELLIFIRE_LIGHTS
         )
         return
+    LOGGER.debug("Disabling Lights - IntelliFire device does not appear to have one")
```

Comment:
I'd invert the check above,log this debug log and `return` if true. Then put the entity addition outdented below.